{"name": "Unsuccessful login", "status": "passed", "description": "Checking successful login with incorrect credentials", "steps": [{"name": "validate error name", "status": "passed", "parameters": [{"name": "error_name", "value": "'incorrect'"}], "start": 1679852560578, "stop": 1679852560683}, {"name": "validate error password", "status": "passed", "parameters": [{"name": "error_password", "value": "'incorrect'"}], "start": 1679852560683, "stop": 1679852560762}, {"name": "clicking to login button", "status": "passed", "start": 1679852560762, "stop": 1679852560823}, {"name": "validate error message", "status": "passed", "parameters": [{"name": "error_message", "value": "'Epic sadface: Username and password do not match any user in this service'"}], "start": 1679852560823, "stop": 1679852560858}], "attachments": [{"name": "log", "source": "9df2ec6e-2ab4-43c9-b28c-9819634a10d5-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "error_name", "value": "'incorrect'"}, {"name": "error_password", "value": "'incorrect'"}, {"name": "error_message", "value": "'Epic sadface: Username and password do not match any user in this service'"}], "start": 1679852560578, "stop": 1679852560858, "uuid": "8f4b4450-a1bb-4af0-a6c3-50791a80c971", "historyId": "9a7837802be43f64432b598f9ed30ef8", "testCaseId": "a7c18bc2d0a4e903f0cff3007abfbc12", "fullName": "tests.test_index_page#test_unsuccessful_login", "labels": [{"name": "severity", "value": "blocker"}, {"name": "owner", "value": "Vlad"}, {"name": "suite", "value": "Authorization suite"}, {"name": "tag", "value": "smoke"}, {"name": "tag", "value": "api"}, {"name": "parentSuite", "value": "tests"}, {"name": "host", "value": "DESKTOP-SO90PSP"}, {"name": "thread", "value": "14616-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_index_page"}]}